<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>SelectMethodWindow</class>
  <widget class="QDialog" name="SelectMethodWindow">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>400</width>
        <height>200</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>Select Matching Method</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="title_label">
          <property name="text">
            <string>Select method of matching</string>
          </property>
          <property name="font">
            <font>
              <pointsize>14</pointsize>
              <weight>75</weight>
              <bold>true</bold>
            </font>
          </property>
          <property name="alignment">
            <set>Qt::AlignCenter</set>
          </property>
        </widget>
      </item>
      <item row="1" column="0">
        <widget class="QPushButton" name="manual_button">
          <property name="text">
            <string>Manual</string>
          </property>
          <property name="font">
            <font>
              <pointsize>12</pointsize>
            </font>
          </property>
          <property name="minimumSize">
            <size>
              <width>150</width>
              <height>50</height>
            </size>
          </property>
        </widget>
      </item>
      <item row="1" column="1">
        <widget class="QPushButton" name="automated_button">
          <property name="text">
            <string>Automated</string>
          </property>
          <property name="font">
            <font>
              <pointsize>12</pointsize>
            </font>
          </property>
          <property name="minimumSize">
            <size>
              <width>150</width>
              <height>50</height>
            </size>
          </property>
        </widget>
      </item>
    </layout>
  </widget>
  <resources/>
  <connections/>
</ui>